encord/objects/ontology_object_instance.py

```diff
@@ -480,6 +547,11 @@ def get_annotation(self, frame: Union[int, str] = 0) -> Annotation:
         Raises:
             LabelRowError: If the frame is not present in the label row.
         """
+        if self._range_only and frame != 0:
+            raise LabelRowError(
+                "This Object Instance only works on ranges, technically only has one 'frame'"
```

Comment:
I appreciate you copied this from my classifications code, but maybe something along the lines of 

`This annotation data for this object instance is stored on only one "frame". Use ....`